### PR TITLE
The clown now bleeds colorful reagent on April Fools.

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

https://github.com/tgstation/tgstation/pull/50815

The clown now bleeds colorful reagent on April Fools.

#### Changelog

:cl:  
rscadd: The clown now bleeds Colorful Reagent on April Fools.
/:cl:

### CODE REVIEW

**Comment by Ratvar** [comment]
Can clown vote rainbow vomit too

**Review by JamieD1 — APPROVED** [review]

**Comment by swissloaf** [comment]
hate turdis, not my fault this time

File: code/modules/mob/living/blood.dm [hunk 1]
```diff
@@ -215,6 +215,8 @@
 /mob/living/carbon/human/get_blood_id()
 	if(HAS_TRAIT(src, TRAIT_HUSK))
 		return
+	if(SSevents.holidays && SSevents.holidays[APRIL_FOOLS] && mind && mind.assigned_role == "Clown")
```
Comment (alexkar598) [inline code/modules/mob/living/blood.dm]:
```suggestion
	if(SSevents.holidays && SSevents.holidays[APRIL_FOOLS] && mind?.assigned_role == "Clown")
```

**Review by alexkar598 — DISMISSED** [review]

**Comment by Marmio64** [comment]
i'd rather have clown bleed colorful reagent all the time

**Review by Hopekz — APPROVED** [review]
Verified Alex's changes were performed. I can't see any other reason code-wise to not perform this merge.


> i'd rather have clown bleed colorful reagent all the time

Make a PR for this if you'd like. This is outside the scope of this PR

**Comment by Hopekz** [comment]
Apparently our new system forces us to dismiss a review instead of approving it to perform a merge so doing that. Fake "dismissal" as it was already addressed.

**Pull request merged**